cmd/pipeline/main.go

```diff
@@ -810,16 +810,33 @@ func main() {
 					appkit.EndpointLogger(commonLogger),
 				)
 
-				integratedservicedriver.RegisterHTTPHandlers(
-					endpoints,
-					clusterRouter.PathPrefix("/features").Subrouter(),
-					errorHandler,
-					kitxhttp.ServerOptions(httpServerOptions),
-					kithttp.ServerErrorHandler(emperror.MakeContextAware(errorHandler)),
-				)
+				{
+					integratedservicedriver.RegisterHTTPHandlers(
+						endpoints,
+						clusterRouter.PathPrefix("/services").Subrouter(),
+						errorHandler,
+						kitxhttp.ServerOptions(httpServerOptions),
+						kithttp.ServerErrorHandler(emperror.MakeContextAware(errorHandler)),
+					)
 
-				cRouter.Any("/features", gin.WrapH(router))
-				cRouter.Any("/features/:featureName", gin.WrapH(router))
+					cRouter.Any("/services", gin.WrapH(router))
+					cRouter.Any("/services/:serviceName", gin.WrapH(router)) // NOTE: parameter name does not matter here
```

Comment:
I think it does, and it should. See my comments bellow.

Reply:
Now it shouldn't anymore.